src/virustotal.ts

```diff
@@ -42,7 +61,7 @@ export class VirusTotal {
     });
 
     return this.instance
-      .post('/files', fd.getBuffer(), {
+      .post(this.largeURL, fd.getBuffer(), {
```

Comment:
As stipulated in their doc "Each upload URL can be used only once." so we should call `getURL` instead.

We should also get a URL for uploading large files only if file size > 32MB, not every time as we are going to quickly hit the rate limit.